client/src/plugins/update-checks/UpdateChecks.js

```diff
@@ -335,16 +366,49 @@ export default class UpdateChecks extends PureComponent {
     this.onClose();
   };
 
+  openVersionInfoPage = () => {
+    this.setState({
+      showModal: true,
+    });
+  };
+
   render() {
     const {
       showModal,
       latestVersionInfo,
       currentVersion,
-      updateChecksEnabled
+      updateChecksEnabled,
+      open,
+      updateAvailable
     } = this.state;
 
+    const {
+      toggle,
+      _buttonRef: buttonRef,
+      close,
+      openVersionInfoPage,
+    } = this;
     return (
       <React.Fragment>
+        {updateAvailable && <Fill slot="status-bar__app" group="9_update_checks">
+          <button
+            style={ {
```

Comment:
Why are we using inline styles here?

Reply:
The footer element styles are inline (talking about Saved button and 💚 icon). I tried creating .less file to add styles initially, but it wasn't working so by looking at the reference buttons, I assumed it's not supposed to work this way. If we can eliminate the inline styles and use separate files for styles in Fill component, then we have to do it across all elements for consistency.

Reply:
We could simply add a `btn--primary` class to `StatusBar.less` and use it for the button.